CONTRIBUTING.md

```diff
@@ -47,19 +47,16 @@ Use [google standards](https://google.github.io/styleguide/pyguide.html)
 
 ### Linter
 
-* Use `flake8` in your IDE
-* Use `pylint` from command line (as in style guide)
+Use the linter of your choice.
+The code style will be checked with [`black`](https://black.readthedocs.io/en/stable/) in the testing.
+We like to use [`flake8`](https://flake8.pycqa.org/en/latest/) and/or [`pylance`](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance).
 
 ### Checklist for code contributions:
   - [ ] Branch off of `main` and name the branch `feature-XX` or `issue-XX`
```

Comment:
We haven't been too strict on this

Reply:
Yeah, I'm good removing. @markbandstra @cosama ?

Reply:
I'm fine with removal.

Reply:
me too.

Reply:
Done